examples/answerer.js

```diff
@@ -111,6 +111,10 @@ class Answerer {
                     }
                 }
             } else {
+                if(candidate && !candidate.candidate) {
+                    //firefox special case, candidate with null candidate field
+                    return;
```

Comment:
Wondering if there's a better way we can handle this one. Can this just move to the same else/if on line 113?